smart-contracts/package.json

```diff
@@ -27,13 +27,20 @@
     "zos-lib": "^2.1.2"
   },
   "devDependencies": {
+    "eslint": "^5.13.0",
```

Comment:
I think eslint should already be in there from the root of the project (we copy some of the dependencies used in all subfolders from there to avoid re-installing the same modules over and over)

Reply:
I'm not sure what the case is in the smart contracts, but when we split off the paywall there was some configuration required to get `link-parent-bin` to copy eslint as expected. These PRs contain those changes just for reference: https://github.com/unlock-protocol/unlock/pull/1511 https://github.com/unlock-protocol/unlock/pull/1525

Reply:
@julien51 would it be okay to create an issue to make this change later?  I cannot use this approach.  Lint does not work in the unlock-app for me, presumably because of #1277 (which I asked for help with).  If we do this, it's just going to get in my way for future changes.

Reply:
Hmm on second thought, I should be able to get it running with a global instance. I can't test locally but I can fire builds at Travis to guess and check. I'll give it a shot.

Reply:
Done.  It just works in Travis, interesting..